src/services/scriptingservice.h

```diff
@@ -172,6 +172,12 @@ class ScriptingService : public QObject {
     Q_INVOKABLE QVariant getApplicationSettingsVariable(
         const QString &key, const QVariant &defaultValue = QVariant());
 
+    Q_INVOKABLE QList<int> fetchSubFoldersByParentId(int parentId);
```

Comment:
Please always use `NoteSubFolder` instead of `SubFolder` and we need documentation for those new methods.